tests/test_config.py

```diff
@@ -41,28 +50,27 @@ def test_legacy_args(tmp_path):
     assert config.rope_condense_ratio == 2
 
 
-def test_from_hf_name():
+@pytest.mark.parametrize("config", config_module.configs, ids=[c["name"] for c in config_module.configs])
+def test_from_hf_name(config):
     from lit_gpt import Config
 
     # by short-hand name
-    config0 = Config.from_name("tiny-llama-1.1b")
+    config0 = Config.from_name(config["name"])
     # or by huggingface hub repo name
-    config1 = Config.from_name("TinyLlama-1.1B-intermediate-step-480k-1T")
+    config1 = Config.from_name(config["hf_config"]["name"])
```

Comment:
this explicitly tests with two different inputs, can you leave it like that please? Otherwise the test won't make any sense

Reply:
The test makes sense when `config["name"]` and `config["hf_config"]["name"]` are different, like in the case with TinyLlama. So this might be a safe net for the future configs, where short and hf names are different.
For the case when these names are equivalent, yes, it doesn't make a lot of sense (apart from showing the issue with shallow copy of nested dicts, but it's more like a coincidence).

So it's better to leave this test as is and in the future, if we add more configs with different short and hf names, change the test to a parametrized one?

Reply:
Actually, I agree. This is an unnecessary complication (my favorite 😊).
If it works in a single case, when a short and hf names are different, it should work in other cases too.

I reverted changes.